website/docs/faq.md

```diff
@@ -17,6 +17,20 @@ To patch vulnerabilities for applications, you can package these applications an
 
 If you find that your storage is rapidly being taken up after working with Copa, run `docker system prune`. This will prune all unused images, containers and caches. 
 
+## How does Copa determine what tooling image to use?
+
+All images being passed into Copa has their versioning data carefully stripped so that an appropriate tooling image can be obtained from a container repository.
```

Comment:
```suggestion
All images being passed into Copa have their versioning data carefully stripped so that an appropriate tooling image can be obtained from a container repository.
```